x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/persistence_registry_uncommon.json

```diff
@@ -67,5 +67,5 @@
   "timeline_title": "Comprehensive Registry Timeline",
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 6
+  "version": 7
```

Comment:
Already bumped `5` to `6` in previous `8.2 Rules` PR: https://github.com/elastic/kibana/pull/129526/files#diff-d4ec3ab4ef485491b700e1ecc3ab83d91940ae66f8e8dd81c5fb538f310c3d20R70